packages/app/src/app/components/StripeMessages/UpgradeSSEToV2.tsx

```diff
@@ -42,7 +42,7 @@ export const UpgradeSSEToV2Stripe = () => {
             window.location.href = sandboxV2Url;
           } else {
             const forkedSandbox = await effects.api.forkSandbox(sandboxId, {
-              isV2: true,
+              v2: true,
```

Comment:
Interesting that this wasn't caught!

Reply:
The redirect was hiding it 😄

Reply:
oh you mean by typescript?

Reply:
Yes `typescript`, not so strict for this codebase.